resources/views/base.blade.php

```diff
@@ -36,12 +36,14 @@ function gtag(){dataLayer.push(arguments);}
 
         <script>
             // Listen for events dispatched from Livewire components...
-            Livewire.on('rules-filtered', () => {
-                document.querySelectorAll('pre code.language-diff').forEach((block) => {
-                    hljs.highlightBlock(block);
+            document.addEventListener('DOMContentLoaded', function () {
+                document.addEventListener('rules-filtered', () => {
```

Comment:
I see, so the trick was about adding one event listener for dom, + nested one for a custom event

Reply:
Also setTimeout(callback, 0);

Which ensures the morphing is done and we have new html in the dom.